sdk/src/wasm/webcrypto_validator.rs

```diff
@@ -331,40 +329,40 @@ pub async fn validate_async(alg: &str, sig: &[u8], data: &[u8], pkey: &[u8]) ->
             )
             .await
         }
-        "rs256" => {
-            async_validate(
-                "RSASSA-PKCS1-v1_5".to_string(),
-                "SHA-256".to_string(),
-                0,
-                pkey.to_vec(),
-                sig.to_vec(),
-                data.to_vec(),
-            )
-            .await
-        }
-        "rs384" => {
-            async_validate(
-                "RSASSA-PKCS1-v1_5".to_string(),
-                "SHA-384".to_string(),
-                0,
-                pkey.to_vec(),
-                sig.to_vec(),
-                data.to_vec(),
-            )
-            .await
-        }
-        "rs512" => {
-            async_validate(
-                "RSASSA-PKCS1-v1_5".to_string(),
-                "SHA-512".to_string(),
-                0,
-                pkey.to_vec(),
-                sig.to_vec(),
-                data.to_vec(),
-            )
-            .await
-        }
-        "es256" => {
+        // "rs256" => {
```

Comment:
should this be deleted?

Reply:
The issue is that – by design – the `SigningAlg` enum does not provide the possibility to describe any algorithms that are not officially supported by C2PA. The whole point of this PR is to make it a compile-time error to attempt to specify any other signing algorithm.

That's why the Rs*** algorithms can't really be supported in this model.

Reply:
Marking as resolved per Slack conversation with @mauricefisher64.